File: isegoria/src/main/java/com/eulersbridge/isegoria/models/PhotoAlbum.java
package com.eulersbridge.isegoria.models;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

@Parcel
public class PhotoAlbum {

    @Json(name = "nodeId")
    public int id;

    public String name;
    public String description;
    public String location;

    @Json(name = "thumbNailUrl")
    public String thumbnailPhotoUrl;

}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/models/PhotoAlbum.kt
package com.eulersbridge.isegoria.network.api.models

import android.annotation.SuppressLint
import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@SuppressLint("ParcelCreator")
@Parcelize
data class PhotoAlbum (
    @Json(name = "nodeId") val id: Int = 0,
    val name: String? = null,
    val description: String?,
    val location: String?,
    @Json(name = "thumbNailUrl") var thumbnailPhotoUrl: String?
) : Parcelable